### ISSUE TITLE
Limited ejections/discharges

### ISSUE THREAD

**Issue by Romain672** [hanabi/hanabi.github.io#784]
Hi,
1) 4-Charms done with number when prompt/finesse/double bluff is impossible, and where Bob have 2 or 3 unclued cards is an ejection on Bob.
2) Unknown trash clues done with number when Bob only have two unclued cards is an ejection on Bob.
3) When r5 is globally known, and a 4-Charm with color on r4 is done, and Bob have only 2 or 3 unclued cards, and Bob plays an ejection, that card is writed as r4.

**Comment by pianoblook** [comment]
4 Double Bluff Ejections have been discussed before. Not particularly against it personally, though I suspect it would cause more confusion than it's worth

**Comment by Zamiell** [comment]
i think there's a language barrier here because i dont understand what he is proposing

**Comment by Romain672** [comment]
It's 'conventions' which apply when per example Bob's hand is x-x-(5)-(5) and need to discharge, or got x-x-x-(5) and need to charm.

**Comment by Zamiell** [comment]
is this just a duplicate of #761?

**Comment by Romain672** [comment]
They are linked for sure.
After all actuals interpretations are impossible, I think #761 just add another one which is 'double ejection', while my idea only happen when Bob don't have enough 'slots not saved' to answer the move asked, and make playable the only slot which wouldn't give false info to the team.

But with a second though, would in nearly all cases that would be unecessary, but Cathy got negative playable in slot 1 (else double bluff). So either the unecessary component is on another player, or either it's a Cathy plays an 'ejection' too hum...

**Comment by mmelwen** [comment]
Agree with Zam, I really dont understand the request here

**Comment by Romain672** [comment]
I looked at ~20 replays.
I find 3 spots: I find no clue to give.

The thing is, for that convention to work, you need:
 - some saved/playable cards on a hand (so often, only after early game)
 - a playable on chop on that person (pretty rare, i got 3 spots in ~20 games)
 - a good target for a charm (which look atrocious since early game is finished)
 - or a discharge (but then, you need that clue to be better than just clueing his chop)

I think what kill this convention is the fact that charm and disacharge are way too rare. And since complexity isn't free (for Bob that's easy, for the clue's receiver it isn't), it doesn't seem enough good.

I close.
